lib/Chunk.js

```diff
@@ -50,6 +50,7 @@ class Chunk {
 		this.ids = null;
 		this.debugId = debugId++;
 		this.name = name;
+		this.isRuntimeOnly = false;
```

Comment:
Could you instead of this create a method called `isOnlyRuntime` that behaves as a mix `isOnlyInitial` and `hasRuntime`?